HW-11/homework11(1).py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+
+
+def MyDecFunc(func):
+    def wrap(*args, **kwargs):
+        time = datetime.now()
+        text = f'Open function {func} \n Time: {time}'
+        with open('file_dec', 'w+') as file_dec:
+            file_dec.write(text)
+        func(*args, **kwargs)
```

Comment:
You should return here the result of the `func` call but not the func itself. Replace these two lines with `return func(*args, **kwargs)`

Reply:
What about this comment?